src/quantum/azext_quantum/_params.py

```diff
@@ -9,6 +9,7 @@
 
 def load_arguments(self, _):
     workspace_name_type = CLIArgumentType(options_list=['--workspace-name', '-w'], help='Name of the Quantum Workspace. You can configure the default workspace using `az quantum workspace set`.', id_part=None, required=False)
+    storage_account_name_type = CLIArgumentType(options_list=['--storage_account', '-sa'], help='Name of the storage account to be used by a quantum workspace.')
```

Comment:
Am I understanding correctly that things like `-g` (for resource group) and `-l` (for location) are handled by some common Azure CLI code? So the only things that need to be listed here are things specific to `az quantum`?

Reply:
Right. Some arguments are inherited from the core `az cli` code.